detection_rules/etc/version.lock.json

```diff
@@ -1576,6 +1618,18 @@
     "type": "eql",
     "version": 109
   },
+  "2c6a6acf-0dcb-404d-89fb-6b0327294cfa": {
+    "rule_name": "Potential Foxmail Exploitation",
+    "sha256": "677b62dc3502ba3192802220e5c25de4e44c1c068cc4cbb54124820c29ce13f2",
+    "type": "eql",
+    "version": 2
+  },
+  "2d62889e-e758-4c5e-b57e-c735914ee32a": {
+    "rule_name": "Suspicious PowerShell Execution via Windows Scripts",
+    "sha256": "a80f52e2d0f126a7c18db7078056274ede0a847de4047bf98ab6fdeb58beef17",
+    "type": "eql",
+    "version": 2
```

Comment:
these new entries but version 1+ may be related to the version lock cleanup when maintaining current-3 I assume?

Reply:
But we also have versions with 1 in new entries, this seems localised.

Reply:
The common integration for the rules with version:2 as new entry is sentinel_one_cloud_funnel

Reply:
For the following new rules
```Potential Escalation via Vulnerable MSI Repair,
Suspicious PowerShell Execution via Windows Scripts,
Potential Foxmail Exploitation,
MsiExec Service Child Process With Network Connection
Suspicious Windows Powershell Arguments
Suspicious Windows Command Shell Arguments
```
which use sentinel_one_cloud_funnel, there is no mintsack, this integration should be minstacked to 8.13